packages/keyring-api/src/rpc-handler.ts

```diff
@@ -47,22 +47,22 @@ async function dispatchRequest(
   assert(request, JsonRpcRequestStruct);
 
   switch (request.method) {
-    case KeyringRpcMethod.ListAccounts: {
+    case `${KeyringRpcMethod.ListAccounts}`: {
```

Comment:
I'm not following on why this change is necessary, from a first look it seems redundant

Reply:
`eslint was complaining here, most likely cause `request.method` is being infered as `string` (which makes sense given that `JsonRpcRequestStruct` is using `method: string()`.


For reference:
- https://typescript-eslint.io/rules/no-unsafe-enum-comparison/

I assume it's considered unsafe to compare a `string` with an `enum` value here, hence the use of template literal to go from (type-wise) `string === enum` to `string === string`

Reply:
That's right, the type of the `request.method` field is `string` but the `case` values were `KeyringRpcMethod`, which caused the linter to complain.

I tought about changing the type of the `method` field, but then the linter would complain about the `default` case.

Another option I tried was to change this assertion:

```ts
assert(request, JsonRpcRequestStruct);
```

To something like (not the exact syntax):

```ts
assert(request, GetAccountRequestStruct | CreateAccountRequestStruct | ...);
```

But the type would not resolve correctly (I'm not sure if it's a Superstruct or TypeScript bug).

Reply:
Interesting, I would assume that the type of an `enum` member to be a specific type (`string`, `number`, `boolean`, etc). It does seems to be confusing but in a minor scale, it can be improved in a future PR.